docs/4_4_1_can.adoc

```diff
@@ -584,11 +588,37 @@ If a Network FMU does not support wake-up this operation can be ignored on consu
 
 |====
 
+===== Network Parameters [[can-network-parameters]]
+Using structural parameters, FMUs can be parameterized according to importer specifications.
+This chapter specifies the structural parameters that each CAN specific FMU shall provide.
+
+====== Bus Notification Parameter [[bus-notification-parameter]]
+For a detailed simulation, the CAN bus behavior regarding acknowledgment, bus errors and arbitration losses must be considered.
+A Bus Simulation can simulate this effect by sending bus notifications in terms of <<ConfirmOpCodeCanLowCut, Confirm>>-, <<BusErrorOpCodeCanLowCut, Bus Error>>- and <<ArbitrationLostOpCodeCanLowCut, Arbitration Lost operations>> to the Network FMUs.
+However in cases where Network FMUs are connected directly or if a Bus Simulation does not simulate such effects, a Network FMU would not receive these operations.
+
+In order to inform Network FMUs not to rely on bus notifications, the importer can set the `org.fmi-standard.fmi-ls-bus.WaitForBusNotification` parameter to `false`, which also shall be the default value.
+Only if the Bus Simulation either supports <<ConfirmOpCodeCanLowCut, Confirm>>-, <<BusErrorOpCodeCanLowCut, Bus Error>>- or <<ArbitrationLostOpCodeCanLowCut, Arbitration Lost operations>>, `org.fmi-standard.fmi-ls-bus.WaitForBusNotification` shall be set to `true`.
```

Comment:
Should a detailed Bus Simulation support both? Why have the case of one, the other, or both?

Reply:
I think the best idea is to discuss that on the F2F Design Meeting next week in present. I already prepared a few slides for that. Is this OK for you @andreas-junghanns?

Reply:
Proposal to new line 601: 
If `org.fmi-standard.fmi-ls-bus.WaitForBusNotification` is set to `true`, the FMU must receive and react to <<ConfirmOpCodeCanLowCut, Confirm>>-, <<BusErrorOpCodeCanLowCut, Bus Error>>- and<<ArbitrationLostOpCodeCanLowCut, Arbitration Lost operations>>.

Reply:
This will be handled in another PR. See [issue comment](https://github.com/modelica/fmi-ls-bus/issues/20#issuecomment-1562744823).